### PR TITLE
go.mod: bump Pebble to 8079611f00bc

### PR DESCRIPTION
**Pull request by jbowens**
Changes:

 * [`8079611f`](https://github.com/cockroachdb/pebble/commit/8079611f) sstable: fix colblk copyDataBlocks off-by-one
 * [`4b36ca34`](https://github.com/cockroachdb/pebble/commit/4b36ca34) db: set FormatNewest to FormatColumnarBlocks
 * [`b57c91c4`](https://github.com/cockroachdb/pebble/commit/b57c91c4) db: add FormatColumnarBlocks
 * [`52888b40`](https://github.com/cockroachdb/pebble/commit/52888b40) db: add TestIteratorSeekPrefixGERandomized
 * [`df9d97d3`](https://github.com/cockroachdb/pebble/commit/df9d97d3) colblk: fix PrefixBytesBuilder.UnsafeGet
 * [`af5365b6`](https://github.com/cockroachdb/pebble/commit/af5365b6) sstable: use specific block reading methods in Describe
 * [`2b3795cd`](https://github.com/cockroachdb/pebble/commit/2b3795cd) sstable: make CopySpan() support columnar blocks
 * [`453b46ca`](https://github.com/cockroachdb/pebble/commit/453b46ca) sstable: remove describe-binary
 * [`80619174`](https://github.com/cockroachdb/pebble/commit/80619174) sstable: cleanup readBlock usage
 * [`0ac9ddc9`](https://github.com/cockroachdb/pebble/commit/0ac9ddc9) crdbtest: move cockroach colblk.KeySchema
 * [`79c9cbfc`](https://github.com/cockroachdb/pebble/commit/79c9cbfc) db: use EncodeSpan in TestCheckLevelsCornerCases
 * [`4b62ea08`](https://github.com/cockroachdb/pebble/commit/4b62ea08) colblk: more reader->decoder renames
 * [`0501caf7`](https://github.com/cockroachdb/pebble/commit/0501caf7) db: consolidate datadriven DB opts parsing; fix block-size bug
 * [`fb2c2ff4`](https://github.com/cockroachdb/pebble/commit/fb2c2ff4) colblk: use uint32 in bundleCalc

Release note: none.
Epic: none.

### CODE REVIEW

**Review by aadityasondhi — APPROVED** [review]

**Comment by RaduBerinde** [comment]
We should hold off on the merge until we figure out the recent regression caused by a Pebble bump.

**Comment by jbowens** [comment]
I'm going to merge this to unblock #132682 now that we've identified the issue. We'll follow up with a subsequent pebble bump with the incoming fix.

bors r=aadityasondhi

**Pull request merged**
